definitions/cloud_services.yaml

```diff
@@ -0,0 +1,27 @@
+# Cloud service artifacts.
+
+name: CloudStorageClients
```

Comment:
This is confusing when there is only one here, lets just create it when we have more than one.